docs/src/index.md

```diff
@@ -8,6 +8,77 @@ For more illustrations of its usage in conjunction with other packages, the
+    - [ScikitLearn.jl](https://scikitlearnjl.readthedocs.io/en/latest/): A Julia wrapper around the full Python scikit-learn machine learning library. Not well integrated with DataFrames, but easily used using statsmodels.jl.
+    - Deep learning: [KNet.jl](https://denizyuret.github.io/Knet.jl/stable/tutorial/#Introduction-to-Knet-1) and [Flux.jl](https://github.com/FluxML/Flux.jl). 
+- **Plotting**
+    - [`Plots.jl`](http://docs.juliaplots.org/latest/): Powerful, modern plotting library with a syntax akin to that of `matplotlib` (in Python) or `plot` (in R). [`StatsPlots`](http://docs.juliaplots.org/latest/tutorial/#Using-Plot-Recipes-1) provides `Plots.jl` with recipes for many standard statistical plots.
+    - [`Gadfly.jl`](http://gadflyjl.org/stable/): High-level plotting library with a "grammar of graphics" syntax akin to that of `ggplot` (in R).
+    - [`VegaLite.jl`](https://www.queryverse.org/VegaLite.jl/stable/): High-level plotting library that uses a different "grammar of graphics" syntax and has an emphasis on interactive graphics.
+- **Data Wrangling**:
+    - [`DataFramesMeta.jl`](https://github.com/JuliaData/DataFramesMeta.jl): A range of convenience functions for DataFrames that augment `select` and `transform` to provide a user experience similar to that provided by `dplyr` in R.
+    - [`Query.jl`](https://github.com/queryverse/Query.jl): `Query.jl` provides a single framework for data wrangling that works with a range of libraries, including DataFrames, other tabular data libraries (more on those below), and even non-tabular data. Provides many convenience functions analogous to those in `dplyr` or `linq`.
+- **And More!**
+    - [`LightGraphs.jl`](https://github.com/JuliaGraphs/LightGraphs.jl): A pure-Julia, high performance network analysis library. Edgelists in DataFrames can be easily converted into Graphs using the [`GraphDataFrameBridge.jl`](https://github.com/JuliaGraphs/GraphDataFrameBridge.jl) package.
+- **IO**:
+    - DataFrames work well with a range of formats, including CSVs (using [`CSV.jl`](https://github.com/JuliaData/CSV.jl)), Stata, SPSS, and SAS files (using [`StatFiles`](https://github.com/queryverse/StatFiles.jl)), and reading (though not writing) parquet files (using [`ParquetFiles`](https://github.com/queryverse/ParquetFiles.jl)).
+
+While not all of these libraries are tightly integrated with DataFrames, because
+DataFrames are essentially collections of aligned Julia vectors, so it is easy
+to (a) pull out a vector for use with a non-DataFrames-integrated library, or (b)
+use `statsmodels` to convert your table into a homogenously-typed matrix.
```

Comment:
```suggestion
While not all of these libraries are tightly integrated with DataFrames, because
DataFrames are essentially collections of aligned Julia vectors, so it is easy
to (a) pull out a vector for use with a non-DataFrames-integrated library, or (b)
convert your table into a homogenously-typed matrix using `Tables.matrix` or StatsModels.jl.
```